mitodl/open-discussions
# Filter out removed/deleted posts and comments from search

#### Pre-Flight checklist

#### What are the relevant tickets?
Fixes #1597

#### What's this PR do?
Filters out removed or deleted posts/comments from search results.

#### How should this be manually tested?
- Perform a search that returns both posts and comments
- Remove one of those posts, and delete another. Ditto for comments.
- Perform the search again.  The deleted and removed posts/comments should no longer show up, but everything else should.
- Perform a search for profiles that you know exist, you should get results.
